packages/frontend/components/SoundCardList.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import SoundCard from './SoundCard'
+import styles from '../styles/SoundBoard.module.scss'
+
+export default function SoundCardList({ list }) {
```

Comment:
It would also be great if you could add a type for the `list` variable, for example you could create an interface for the type

Reply:
done. But I wanted to separate the list of available sound to its own file. So it is just a .ts file that doesn't have any react component in it, just some helper variable. Where should I put it? Currently I put it in /components/soundList.ts

Reply:
If it's only used from within the component (won't be used anywhere else in the project), I think it's better to keep it in the same file as the component

Reply:
How about the interface? I used in another file too

Reply:
I usually put all interfaces in separate folder beside the `components` folder, e.g: `interfaces` folder
and then in It you can create a new ts file to save the interface

Reply:
Ok done